src/test/java/seedu/address/logic/commands/ListCommandTest.java

```diff
@@ -41,21 +46,21 @@ public void execute_listIsNotFiltered_showsSameList() {
     }
 
     @Test
-    public void execute_listIsFiltered_showsLectures() {
+    public void execute_listIsFiltered_moduleFoundShowLectures() {
         ModuleCode moduleCode = module.getCode();
         String input = String.format("list /mod %s", moduleCode);
         try {
             listCommand = new ListCommandParser().parse(input);
             listCommand.execute(expectedModel);
-        } catch (ParseException e) {
+        } catch (Exception e) {
             assertNull(e);
         }
```

Comment:
Just a suggestion, but in such cases where exceptions are not suppose to be expected, can just add a `throws Exception` to the method header so you don't have to write this catch block. I feel it'll make the code easier to read, and if an unexpected exception is thrown and is not caught, the test case will fail.